### PR TITLE
Create msm8930-common.mk

### PR DESCRIPTION
**Pull request by ghost**
This time it's more OK. (I hope so)

### CODE REVIEW

File: BoardConfigCommon.mk [hunk 1]
```diff
@@ -60,7 +62,6 @@ BOARD_KERNEL_CMDLINE := \
     msm_rtb.filter=0x3F \
     ehci-hcd.park=3 \
     androidboot.bootdevice=msm_sdcc.1
-BOARD_MKBOOTIMG_ARGS := --ramdisk_offset 0x02000000
```
Comment (feherneoh) [inline BoardConfigCommon.mk:63]: Why would we remove this one?

Comment (ghost) [reply BoardConfigCommon.mk:63]: Read the note above. This flag is included in the mkbootimg.mk. You will have to merge the boot logo into Lumia 720 device tree in order to make this working.
And why you did not move the init library into the common tree?

Comment (feherneoh) [reply BoardConfigCommon.mk:63]: That flag is supported in the original boot image generation makefile too, it has nothing to do with the custom one

**Comment by ghost** [comment]
I have only applied @r3pwn 's patches. And he did it in that way. Of course you can merge it as you want.
P.S.: You have an experimental branch in your own LK repo. Can a bootloader compiled from this branch boot? And what's about @gabixdev 's L520_stable branch?

**Comment by feherneoh** [comment]
my experimental LK branch is for LKLoader, not aboot-LK
LK wasn't changed in any positive way for now, and I won't even touch that unless we really need to fix someting in it. The new bootloader is NOT LK, and is not uploaded anywhere yet. For that, I have to finish writing the display driver first.

**Comment by ghost** [comment]
EFIDroid?

The following commits could move the init library to the common repo

**Comment by feherneoh** [comment]
Init library was already moved.
https://github.com/Android4Lumia/android_external_libinit_dpp
Forget this repo until I push my final changes, after that it will be way easier to suggest changes those will actually be accepted.

**Comment by ghost** [comment]
I thought it would be better if it is moved into the common device tree instead of an external library because it's device specific.
And what do you mean with the new kernel?

**Comment by feherneoh** [comment]
It's not device specific, and definitely not Lumia-8930 specific. It can support any device that has DPP, so basically any WP8+ device.

**Pull request closed without merging**